web/app/widgets/image/image.widget.js

```diff
@@ -39,9 +39,24 @@
     function ImageController ($rootScope, $scope, OHService, $interval) {
         var vm = this;
         this.widget = this.ngModel;
+
+        function updateValue() {
+            var item = OHService.getItem(vm.widget.item);
+            if (!item || vm.widget.image_source !== 'item' || item.type !== "String") {
+                vm.value = "";
+                return;
+            }
+            vm.url = $sce.trustAsResourceUrl(item.state);
+        }
+
+        OHService.onUpdate($scope, vm.widget.item, function () {
+            updateValue();
+        });
         
-        vm.original_url = vm.url = this.widget.url;
-        var intervaltype = vm.intervaltype = this.widget.intervaltype || 'seconds';
+        if (this.widget.image_source === 'static') {
```

Comment:
`image_source` should be defaulted to `static` then because this breaks existing configs (where `image_source` is undefined).